common/context.go

```diff
@@ -9,6 +9,7 @@ type ctxKey int
 const (
 	ctxKeyLaunchOptions ctxKey = iota
 	ctxKeyHooks
+	ctxKeyTraceID
```

Comment:
Should we be consistent with the logger field name?

```suggestion
	ctxKeyIterationID
```

Reply:
Ah, thought I renamed them all. Sure :+1:

Reply:
Resolved by a65d830.